templates/spec/components/breadcrumbs_component_spec.rb

```diff
@@ -0,0 +1,71 @@
+require "solidus_starter_frontend_helper"
+
+# I'm getting the "NameError Exception: uninitialized constant
+# `#<Class:0x000056478b9a4100>::ActiveStorageAttachment`" in byebug unless I
+# require spree/taxon.
+require 'spree/taxon'
```

Comment:
Highlighting the comment: I'm getting the "NameError Exception: uninitialized constant `#<Class:0x000056478b9a4100>::ActiveStorageAttachment`" in byebug unless I require spree/taxon.